packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -279,26 +279,26 @@ contract L2OutputOracle is Initializable, ISemver {
     }
 
     /**
-     * @notice Updates the latest finalized output index. This function may only be called by the
+     * @notice Updates the next finalize output index. This function may only be called by the
```

Comment:
How about `next finalized output index` or `next output index to be finalized`, instead of `next finalize output index`?